Package.swift

```diff
@@ -1,30 +1,32 @@
-// swift-tools-version:4.0
+// swift-tools-version:5.0
 
 import PackageDescription
 
 let package = Package(
   name: "PactConsumerSwift",
+  platforms: [
+    .macOS(.v10_10), .iOS(.v8), .tvOS(.v9)
+  ],
   products: [
-    .library(
-      name: "PactConsumerSwift",
-      targets: ["PactConsumerSwift"]
-    ),
+    .library(name: "PactConsumerSwift", targets: ["PactConsumerSwift"])
   ],
   dependencies: [
-    .package(url: "https://github.com/Alamofire/Alamofire.git", from: "4.6.0"),
-    .package(url: "https://github.com/Quick/Quick.git", from: "1.1.0"),
-    .package(url: "https://github.com/Quick/Nimble.git", from: "7.0.0"),
-    .package(url: "https://github.com/Thomvis/BrightFutures.git", from: "6.0.0")
+    .package(url: "https://github.com/Quick/Nimble.git", from: "8.0.0"),
+    .package(url: "https://github.com/antitypical/Result.git", from: "5.0.0"),
+    .package(url: "https://github.com/Quick/Quick.git", from: "2.1.0"),
+    .package(url: "https://github.com/SwiftyJSON/SwiftyJSON.git", from: "5.0.0"),
+    .package(url: "https://github.com/Thomvis/BrightFutures.git", from: "8.0.0"),
+    .package(url: "https://github.com/nonec/SwiftPactServer.git", from: "1.0.0"),
```

Comment:
Instead of having yet another git repo you rely on due to SPM, you could have it as part of the current project and define dependency as `.package(path: "path/to/folder/containing/SwiftPactServer")`

Reply:
I remember thinking about that. I think part of the problem was having big files in that and I couldn't use git-lfs on a fork because that would count into the space of the origin.
I'm not a 100% sure, but I think this would be something to resolve if we ever get this to a mergeable state.